appinventor/blocklyeditor/src/blocklyeditor.js

```diff
@@ -156,6 +157,7 @@ Blockly.BlocklyEditor.render = function() {
   var timeDiff = stop - start;
   Blockly.Instrument.stats.totalTime = timeDiff;
   Blockly.Instrument.displayStats("Blockly.BlocklyEditor.render");
+  Blockly.WarningHandler.checkAllBlocksForWarningsAndErrors();
```

Comment:
Why is this change here? This adds significant computation to an inner loop function. This can significantly slow down projects with a lot of blocks.